### ISSUE TITLE
[Bug Report] Migration failed: Unexpected token < in JSON

### ISSUE THREAD

**Issue by LenAnderson** [stashapp/stash#2197]
**Describe the bug**
When trying to perform schema migration from schema version 28 to 29 it fails after a while with the error message.

> Unexpected token < in JSON at position 0

**To Reproduce**
Update stashapp and perform schema migration (in my case from 28 to 29).

**Expected behavior**
Successful schema migration

**Screenshots**
![Screenshot](https://i.imgur.com/RJO7UdV.png)

**Stash Version: (from Settings -> About):**
stash version: v0.12.0 (fb864a6e - Official Build) - 2021-12-28 23:26:19

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Google Chrome
 - Version: 96.0.4664.110 (Official Build) (64-bit)

**Additional context**
Stash is running as a docker container on Unraid 6.9.2.
Authentication (login with user+pw) is enabled.

**Comment by kermieisinthehouse** [comment]
Most users have migrated without issue. To help us debug this, can you confirm that
- refreshing this page doesn't show that the migration is already done
- a simple retry doesn't fix it

Then, can you open devtools, go to the network tab, and run the migration, and post the response data that you see? Also, are you running Stash behind a proxy that enforces a timeout?

**Comment by LenAnderson** [comment]
I guess the proxy timeout may have been the issue. I'm running Stash behind nginx proxy manager. While I can't find anything regarding timeouts in the proxy settings a default timeout may still apply.

I did try to start the migration a couple of times, both by clicking the button again when the error message shows up as well as refreshing the page and trying again. However, now that I come back to it 15 minutes later Stash loads without the migration message and is on v0.12.0.


Ok, there's actually an open issue on nginx proxy manager about this. Looks like the only way to increase timeout is a custom / advanced config.  
https://github.com/jc21/nginx-proxy-manager/issues/257


Thanks for the help :) I suppose that solves it. Closing the issue.
